e2e/claims.service.e2e.ts

```diff
@@ -25,11 +25,11 @@ const { namehash } = utils;
 
 const provider = new providers.JsonRpcProvider(rpcUrl);
 const staticIssuer = Wallet.createRandom().connect(provider);
-const staticIssuerDID = `did:${Methods.Erc1056}:${staticIssuer.address}`;
+const staticIssuerDID = `did:${Methods.Erc1056}:volta:${staticIssuer.address}`;
```

Comment:
You can import chain name from https://github.com/energywebfoundation/ew-did-registry/blob/ac62f5913cc71cb443b47c7b38e92f818eb502f9/packages/did/src/models/index.ts#L12

Reply:
I'll make this change